### PR TITLE
Create secureprofiles.tag

### PR DESCRIPTION
**Pull request by Kas-tle**
Tag for secure profiles and how to disable them.

### CODE REVIEW

File: src/main/resources/tags/errors/secureprofiles.tag [hunk 1]
```diff
@@ -0,0 +1,16 @@
+type: text
+issues: This server requires secure profiles
+
+---
+
+Minecraft 1.19 adds key signing, which is currently only used to verify that chat messages purported to be sent by a user come from said user by verification through the user's Mojang-signed public key.
```
Comment (Konicai) [inline src/main/resources/tags/errors/secureprofiles.tag:6]:
```suggestion
Minecraft 1.19 adds key signing, which is currently only used to verify that chat messages are coming from the right player. 
```

**Review by Konicai — COMMENTED** [review]
Although I think what you currently have is perfectly concise for developers and stuff, this will definitely go over the target audience's head

**Comment by Kas-tle** [comment]
Fair enough.

**Review by Konicai — APPROVED** [review]

File: src/main/resources/tags/errors/secureprofiles.tag [hunk 2]
```diff
@@ -0,0 +1,16 @@
+type: text
+issues: This server requires secure profiles
```
Comment (Konicai) [inline src/main/resources/tags/errors/secureprofiles.tag:2]:
```suggestion
issues: This server requires secure profiles || multiplayer.disconnect.missing_public_key
```

**Review by Konicai — APPROVED** [review]

**Review by RednedEpic — APPROVED** [review]

**Pull request merged**